ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_facts.py

```diff
@@ -581,6 +581,13 @@ def platform_settings(self):
 
         return {}
 
+    @cached_property
+    def vtep_vvtep_ip(self):
+        """
+        switch.vtep_vvtep_ip set based on Global var vtep_vvtep_ip
+        """
+        return get(self._hostvars, "vtep_vvtep_ip", default=None)
```

Comment:
None is already the default value returned by get so no need to add it here
```suggestion
        return get(self._hostvars, "vtep_vvtep_ip")
```